tests/registry.cc

```diff
@@ -90,19 +90,19 @@ test_RegistryFileMixinGetRegWithoutFile()
 /* -------------------------------------------------------------------------- */
 
 bool
-test_RegistryFileMixinEmptyPath()
+test_ManifestFileMixinEmptyPath()
 {
   using namespace flox;
 
-  flox::command::RegistryFileMixin rfm;
-  // Try loading the registry without setting the path
+  flox::command::ManifestFileMixin rfm;
+  /* Try loading the registry without setting the path. */
   try
     {
-      rfm.setRegistryPath( "" );
+      rfm.manifestPath = "";
       RegistryRaw regRaw = rfm.getRegistryRaw();
       return false;
     }
-  catch ( FloxException & )
+  catch ( ... )
```

Comment:
TODO: Create a `FloxFileNotFound` exception

Reply:
suggestion(non-blocking): Create an issue for it?

Reply:
I went ahead and added error handling